draft-ietf-rats-corim.md

```diff
@@ -1902,17 +1903,9 @@ Conditional Endorsement Triple Transformation :
 
 * The `cmtype` of the `ev` entry's `addition` ECT is set to `endorsements`.
 
-* For each entry in the Conditional Endorsement Triple ({{sec-comid-triple-cond-endors}}) `conditions` array, the `stateful-environment-record` is copied to a `condition` ECT in the `ev` entry.
-
-* For each `measurement-map` entry in the measurements list of the `stateful-environment-record`, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `condition` ECT `element-map` of the `element-list`.
-
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/303
-
-* For each entry in the Conditional Endorsement Triple `endorsements` array, the `endorsed-triple-record` is copied to an `addition` ECT in the `ev` entry.
-
-* For each `measurement-map` entry in the measurements list of the `endorsed-triple-record`, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `addition` ECT `element-map` of the `element-list`.
+* For each entry in the Conditional Endorsement Triple ({{sec-comid-triple-cond-endors}}) `conditions` list, the `stateful-environment-record` `environment` is copied to the `environment` of the `ev` `condition` ECT and the `stateful-environment-record` `claims-list` entries are copied to the `element-list` of the `ev` `condition` ECT.
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/303
+* For each entry in the Conditional Endorsement Triple `endorsements` list, the `endorsed-triple-record` `condition` `environment-map` is copied to the `environment` of the `ev` `addition` ECT and the `endorsed-triple-record` `endorsement` list entries are copied to the `element-list` entries of the `ev` `addition` ECT.
```

Comment:
A tough nut to parse.  I am wondering whether pseudo-code would be a better fit for describing this kind of stuff.

Reply:
for example, something like (to be checked):
```
foreach e in CET:
  copy(e.endorsed-triple-record.condition.environment-map, ECT.ev.addition.environment)
  copy(e.endorsed-triple-record.entries, ECT.ev.addition.element-list)
```

Reply:
Is dot notation universal enough that readers won't be confused by it? For example, the CDDL RFC says "CDDL allows the specification of a data item type by referring to the
   CBOR representation (specifically, to major types and additional
   information; see [Section 2 of [RFC7049]](https://www.rfc-editor.org/rfc/rfc7049#section-2))."
But I don't think that text is describing this convention.

Reply:
I might favor using back quotes for each CDDL expression so the dot isn't inside of a back quoted section. Such as:

`e`.`endorsed-triple-record`.`condition`.`environment-map`

Reply:
Addressed in commit https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/315/commits/d43b90eff26c3abea4ae8879b0ee47a036612b2e